### PR TITLE
T14014

### PR DESCRIPTION
**Pull request by tchx84**
**NOT READY FOR MERGE YET**

https://phabricator.endlessm.com/T14014

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -59,15 +59,15 @@ done
...
-test -z ${datarootdir} && datarootdir=${prefix}/share
+test -z ${datarootdir} && datarootdir=share
 test -z ${datadir} && datadir=${datarootdir}
 test -z ${sysconfdir} && sysconfdir=${prefix}/etc
-test -z ${libdir} && libdir=${prefix}/lib
-test -z ${mandir} && mandir=${prefix}/share/man
-test -z ${includedir} && includedir=${prefix}/include
+test -z ${libdir} && libdir=lib
+test -z ${mandir} && mandir=share/man
+test -z ${includedir} && includedir=include
```
Comment (ebassi) [inline configure]: This whole chunk is only necessary if you're building with meson ≤ 0.35.0; starting with meson 0.36.0, the paths passed are absolute.

File: debian/libemeus-1.install [hunk 2]
```diff
@@ -0,0 +1,2 @@
+usr/bin/emeus-gen-constraints
```
Comment (ebassi) [inline debian/libemeus-1.install]: This should probably be in a separate package, like: `emeus-1-tools`

File: debian/rules [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --builddirectory=_build
+
+override_dh_auto_configure:
+	mkdir _build && cd _build && ../configure --prefix=/usr
```
Comment (ebassi) [inline debian/rules]: Debian supports multiarch, so it should probably specify the `--libdir` accordingly. I'm not fluent in Debian packaging, though, so you may need to ask Dan.

File: debian/control [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+               gtk-doc-tools (>= 1.18),
+               gobject-introspection,
+               libgirepository1.0-dev (>= 1.30),
+               libglib2.0-dev (>= 2.38),
+               libgtk-3-dev,
+               meson,
+               ninja-build
+Standards-Version: 3.9.4
+Section: free/libs
+Homepage: https://github.com/ebassi/emeus
```
Comment (ebassi) [inline debian/control]: The homepage should be `https://ebassi.github.io/emeus`

File: configure (hunk 1 above)
Comment (tchx84) [inline configure]: What do you recommend? Should we update to meson 0.36? I think you mentioned something about emeus needing 0.34 for some reason, what as it?

**Comment by tchx84** [comment]
As discussed previously, I am closing this PR and moving it to endlessm fork (as it was originally intended)

**Pull request closed without merging**